location.md

```diff
@@ -36,13 +36,19 @@ With three state universities, many tech companies ranging from small startups t
 
 ## Travel
 
-<!-- TODO: Change me -->
-Zürich is well connected to the rest of Europe with many direct flights and trains to other major European cities every day.
+### By train
+
+Paris is very easily reachable by train. There are [seven different train stations](https://en.wikivoyage.org/wiki/Paris#By_train), all of which are connected through the local transport system. Many direct trains are available from various European cities such as Amsterdam, Brussels, Cologne, Frankfurt, London, Milan, Munich, Turin, and Zurich.
```

Comment:
I think only two of these train stations are actually internationnaly reachable and only four are considered as train stations (gare de lyon, gare du nord, gare montparnasse and gare d'austerlitz).

Reply:
Good to know! I decided to highlight the international ones based on this map: https://en.wikipedia.org/wiki/List_of_railway_stations_in_Paris#/media/File:Railway_stations_on_departure_from_Paris_map-en.svg